src/main/java/core/basesyntax/Info.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+interface Info {
+    public void printInfo();
```

Comment:
Usually IDE highlights in a grey color those parts of code which can be deleted. Doesn't your EDI do it with the public key work here?